diff --git a/scriptmodules/emulators/retroarch.sh b/scriptmodules/emulators/retroarch.sh
index 2a28bf4..e1c14a4 100644
--- a/scriptmodules/emulators/retroarch.sh
+++ b/scriptmodules/emulators/retroarch.sh
@@ -40,7 +40,7 @@ function sources_retroarch() {
 
 function build_retroarch() {
     local params=(--disable-sdl --enable-sdl2 --disable-oss --disable-al --disable-jack)
-    ! isPlatform "x11" && params+=(--disable-x11 --disable-ffmpeg --disable-pulse)
+    ! isPlatform "x11" && params+=(--disable-x11 --disable-pulse)
     isPlatform "gles" && params+=(--enable-opengles)
     isPlatform "rpi" && params+=(--enable-dispmanx)
     isPlatform "mali" && params+=(--enable-mali_fbdev)
